specifications/api/swagger.yaml

```diff
@@ -2014,16 +2029,12 @@ components:
     ApiSharedWalletPostData: &ApiSharedWalletPostData
       type: object
       required:
-        - name
-        - account_public_key
-        - account_ix
-        - address_pool_gap
+        - retrieve_method
```

Comment:
:thinking: What is this for ?

Reply:
Okay got it, that's for discriminating. 
Let's not go that way but be consistent with other multi-constructor approaches (i.e. have a flat object with partially overallaping fields).